### ISSUE TITLE
[Question] Can I disable everything and only use EasyClick?

### ISSUE THREAD

**Issue by neumannjan** [pit-ray/win-vind#210]
**Question**
I am an active user of win, mac and linux. I indeed do switch between all three OSes regularly. In general, I try to make my keyboard shortcut setups on all three systems as close as possible, to be able to switch easily. On Mac I use the app called [Homerow](https://www.homerow.app/). It works as follows: with cmd+shift+spacebar you get EasyClick. With cmd+shift+J you get the ability to scroll windows, which goes away when you press Esc. That's all it does.

It seems that win-vind is the only app to offer me the same capability on Windows. However, even though I'm an avid user of vim, I do feel quite overwhelmed by all the capabilities of win-vind. While it seems that it is a super powerful application and I'd probably benefit from learning it eventually, I don't really want to right now. I'm simply used to the workflow I have already, and it gets me 90% to the ideal, while being replicable.

Tl;dr is there a way to just use EasyClick and nothing else?

Thanks.

**Comment by kwonoj** [comment]
@pit-ray This works great, mind if I ask additional questions?

Is there way to control click behavior once entered easyclick mode with certain bindings? I want to either ctrl+click / or normal click once after enabling easyclick, instead of having separate keybindings for easyclick_left vs easyclick_mid (or else).

Also, is there a scroll mode keybindings as well?

**Comment by pit-ray** [comment]
@kwonoj 
I don't know if I caught your question correctly, but now easyclick is an independent command `<easyclick>`, not `<easyclick_left>, etc.
https://pit-ray.github.io/win-vind/cheat_sheet/functions/#easyclick

As with Vim, you can define the user-defined sequential command, which allows you to control the behavior after the `<easyclick>`. For example, `map <s-f><s-f> <easyclick><click_right>` and so on.

However, functions such as long pressing the control key have not yet been implemented. #41, #69.

Thanks.

**Comment by kwonoj** [comment]
@pit-ray Thanks for the answer. I'm trying couple to see if it works as intended, so far it looks like I'm misunderstanding or misconfiguration something.

The basic crux I want to achieve are 

- easyclick allows click with kbd combination:
This option in homerow summarizes the behavior I'm trying to achieve

<img width="324" alt="image" src="https://github.com/pit-ray/win-vind/assets/1210596/5fa4d756-0c29-4eb1-8123-2e97408da7f0">

once easyclick combination is entered, this option allows to trigger click only if certain kbd (space / enter in this case). The kbd input can be combined with modifiers, so for example I grabbed an element in the web browser, then `ctrl-enter` triggers ctrl-click, and normal enter triggers normal click.

- scroll `mode`
a key combination that enables hjkl triggers scroll event until exiting mode.
